catalog/ui/catalog-ui-search/webpack/config/base.js

```diff
@@ -16,7 +16,7 @@ let commitHash = childProcess
     .toString();
 
 let isDirty = childProcess
-    .execSync('git st')
+    .execSync('git status')
```

Comment:
We might want to consider adding a timeout for this ..
https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options